fs/any_version.go

```diff
@@ -27,6 +43,12 @@ func (*AnyVersion) IsSourceImpl() src.InstallSrcSigil {
 }
 
 func (av *AnyVersion) Validate() error {
+	if av.ExactBinPath != "" && (av.Product != nil || len(av.ExtraPaths) > 0) {
+		return fmt.Errorf("use either ExactBinPath or Product + ExtraPaths, not both")
+	}
+	if !filepath.IsAbs(av.ExactBinPath) {
```

Comment:
Should this have a `av.ExactBinPath != ""` check prior to checking for the absolute file path? Sorry thought the testing would've caught this, but for some reason getting this error when implementing v0.3.0 downstream:

```
=== RUN   TestParallelTest
cannot run Terraform provider tests: 1 error occurred:
        * expected ExactBinPath ("") to be an absolute path


FAIL    github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource    1.132s
```

Reply:
Oh, that is a good catch! Let me fix that...

Reply:
Ah sorry, just submitted: https://github.com/hashicorp/hc-install/pull/42